### PR TITLE
Fixed manual precision init for propagation based mode

### PR DESCRIPTION
**Pull request by ljaljushkin**
solves CVS-49035

### CODE REVIEW

**Comment by ljaljushkin** [comment]
Jenkins please retry a build

File: nncf/quantization/precision_init/manual_init.py [hunk 1]
```diff
@@ -39,23 +38,31 @@ class ManualPrecisionInitializer(BasePrecisionInitializer):
...
-                raise ValueError('Invalid format of bitwidth per scope: [int, str] is expected')
-            bitwidth = pair[0]
-            scope_name = pair[1]
+            bitwidth, scope_name = pair
             is_matched = False
-            for scope, quantizer in self._all_quantizers_per_scope.items():
-                if in_scope_list(str(scope), scope_name):
-                    quantizer.num_bits = bitwidth
+            msg = 'Failed to assign bitwidth={} to `{}`,\n' \
+                  'because it is incompatible with supported quantization configs: {}'
```
Comment (AlexKoff88) [inline nncf/quantization/precision_init/manual_init.py:51]: Can you also add in the message something like "for the specified target hardware" for more clarity?

Comment (ljaljushkin) [reply nncf/quantization/precision_init/manual_init.py]: Done

**Comment by ljaljushkin** [comment]
Jenkins please retry a build

**Review by AlexKoff88 — APPROVED** [review]

**Review by alexsu52 — APPROVED** [review]

**Pull request merged**